site/docs/using.md

```diff
@@ -2,7 +2,7 @@
 layout: docs
 menu: docs
 title: Embedding Vega-Lite in a Web Page
```

Comment:
We can use Vega-Lite in command line via vl2png, vl2svg too?   

Should we change title to Using Vega-Lite and add more sections for those?   
(And possibly add in-page TOC on the top of the page?)